src/views/open-api-docs-view/components/open-api-overview/components/description-mdx/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote'
+import {
+	MdxA,
+	MdxOrderedList,
+	MdxUnorderedList,
+	MdxListItem,
+	MdxTable,
+	MdxH1,
+	MdxH2,
+	MdxH3,
+	MdxH4,
+	MdxH5,
+	MdxH6,
+	MdxP,
+	MdxInlineCode,
+	MdxBlockquote,
+	MdxPre,
+} from 'components/dev-dot-content/mdx-components'
+import s from './description-mdx.module.css'
+
+type MdxComponents = Record<string, (props: unknown) => JSX.Element>
+
+const DEFAULT_MDX_COMPONENTS: MdxComponents = {
+	a: MdxA,
+	blockquote: MdxBlockquote,
+	h1: MdxH1,
+	h2: MdxH2,
+	h3: MdxH3,
+	h4: MdxH4,
+	h5: MdxH5,
+	h6: MdxH6,
+	inlineCode: MdxInlineCode,
+	li: MdxListItem,
+	ol: MdxOrderedList,
+	p: MdxP,
+	pre: MdxPre,
+	table: MdxTable,
+	ul: MdxUnorderedList,
+}
+
+/**
+ * TODO: write description
+ */
+export function DescriptionMdx({
```

Comment:
Uses all the same MDX component primitives as `DevDotContent`, but without support for additional custom components, as I think we want our spec file `description` values to stay fully portable and CommonMark-compliant!